docs-src/concepts/what-is-a-timer.md

```diff
@@ -0,0 +1,24 @@
+---
+id: what-is-a-timer
+title: What is a Timer?
+sidebar_label: Timer
+description: A Workflow can set a durable timer for a fixed time.
+tags:
+  - term
+  - explanation
+---
+
+A Workflow can set a durable timer for a fixed time.
+Timers in Temporal are persisted, meaning that even if your Worker or Temporal Cluster is down when the time period completes, as soon as your Worker and Cluster are back up, the `sleep()` or `timer()` call will resolve and your code will continue executing.
+This makes it a reliable and resource-light operation that does not tie up the process, and you can run millions of Timers off a single Worker.
```

Comment:
I wonder if "tie up" and "run ... off" would be clear to non-native English speakers. Consider this alternate phrasing, which may be more clear:

> Timers are reliable and efficient. Workers consume no additional resources while waiting for a Timer to fire, so a single Worker can await millions of Timers concurrently.